keyboards/handwired/sick68/keymaps/default/config.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2020 umbynos
```

Comment:
This file can be deleted.

Reply:
I've used `./util/new_keyboard.sh` utility to generate all the required files, so this file was auto-generated. If It's not useful I'll remove it.